src/System.Drawing.Common/tests/RegionTests.cs

```diff
@@ -0,0 +1,1930 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
```

Comment:
@danmosemsft I didn't ask them about this case in particular. Could we split the file so that whatever came from Mono is in a single file and the new tests in a separate one?